logger/modules/log_file_merger.py

```diff
@@ -0,0 +1,77 @@
+"""
+Merges log files.
+"""
+
+import os
+import pathlib
+from datetime import datetime
+
+from ..read_yaml.modules import read_yaml
+
+CONFIG_FILE_PATH = pathlib.Path(os.path.dirname(__file__), "config_logger.yaml")
+MERGED_LOGS_FILENAME = "merged_logs"
+
+
+def merge_log_files(log_file_directory: str) -> None:
+    """
+    Reads, sorts, and writes log files in the specified directory.
+
+    log_file_directory: The directory containing the log files to be merged.
+    """
+    # Read log files
+    log_files = [
+        file
+        for file in os.listdir(log_file_directory)
+        if file.endswith(".log") and file != f"{MERGED_LOGS_FILENAME}.log"
+    ]
+    log_entries = []
+    for log_file in log_files:
+        with open(os.path.join(log_file_directory, log_file), "r", encoding="utf-8") as file:
+            log_entries.extend(file.readlines())
+
+    # Sort log entries
+    log_entries.sort(key=lambda entry: datetime.strptime(entry.split(": ")[0], "%H:%M:%S"))
+
+    # Write merged logs
+    merged_log_file = os.path.join(log_file_directory, f"{MERGED_LOGS_FILENAME}.log")
+    with open(merged_log_file, "w", encoding="utf-8") as file:
+        file.writelines(log_entries)
+
+
+def get_current_run_directory() -> str:
```

Comment:
Change function signature:
* Parameters: `pathlib.Path` , the path to the config file
* Return: `tuple[bool, pathlib.Path | None]` , success, the directory to the logs of the most recent run